### DISCUSSION TITLE
How to change preflight http method HEAD to GET?

### DISCUSSION THREAD

**Discussion by yuuk** [vercel/next.js#41615] [Help]
![image](https://user-images.githubusercontent.com/9426456/197105774-80ce3758-21c6-40fe-b1ae-aec2e7b77b3b.png)

Our server blocked HEAD http method, It's there some ways to change preflight http method?

**Comment by gaston-flores** [comment]
Unfortunately, you can't. Preflight requests [are sent by the browser](https://developer.mozilla.org/en-US/docs/Glossary/Preflight_request) and can't be controlled from your app. If you want to avoid the browser sending that request, make sure your own request meets [these criteria](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#simple_requests).
